js/stat.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+var CLOUD_WIDTH = 420;
+var CLOUD_HEIGHT = 270;
+var CLOUD_X = 100;
+var CLOUD_Y = 10;
+var SHADOW_GAP = 10;
+var EXTERNAL_GAP = 30;
+var FONT_GAP = 18;
+var GAP = 80;
+var BAR_WIDTH = 40;
+var BAR_GAP = 50;
+var MAX_BAR_HEIGHT = 150;
+var CLOUD_COLOR = '#ffffff';
+var SHADOW_COLOR = 'rgba(0, 0, 0, 0.7)';
+
+var renderCloud = function (ctx, x, y, color) {
+  ctx.fillStyle = color;
+  ctx.fillRect(x, y, CLOUD_WIDTH, CLOUD_HEIGHT);
+};
+
+var renderText = function (ctx, text, gap) {
+  ctx.fillStyle = '#000000';
+  ctx.font = '16px PT Mono';
+  ctx.fillText(text, CLOUD_X + EXTERNAL_GAP, CLOUD_Y + EXTERNAL_GAP + gap);
+};
+
+var getMaxElement = function (arr) {
+  return Math.max.apply(null, arr);
+};
+
+var getColumnColor = function () {
+  var randomNumber = (Math.floor(Math.random() * 100) + 1);
+  return 'hsl(240,' + randomNumber + '%, 50%)';
+};
+
+var drawColumn = function (ctx, x, y, width, height) {
+  ctx.fillRect(x, y, width, height);
+};
+
+var drawHistogram = function (ctx, players, times) {
+  var maxTime = getMaxElement(times);
+
+  players.forEach(function (player, i) {
+    var barHeight = (MAX_BAR_HEIGHT * times[i]) / maxTime;
+    var columnX = CLOUD_X + EXTERNAL_GAP + (BAR_GAP + BAR_WIDTH) * i;
+    var columnY = CLOUD_Y + GAP + (MAX_BAR_HEIGHT - barHeight);
+    var columnTextY = CLOUD_Y + GAP + MAX_BAR_HEIGHT + FONT_GAP;
+
+    ctx.fillText(Math.ceil(times[i]), columnX, columnY - 10);
+
+    // теперь вроде работает только ESLint возмущается
+    player === 'Вы' ? ctx.fillStyle = 'rgba(255, 0, 0, 1)' : ctx.fillStyle = getColumnColor();
```

Comment:
Так тернарник не записывается, нужно так
```suggestion
    // теперь все работает и ESLint больше не возмущается
    ctx.fillStyle = player === 'Вы' ? 'rgba(255, 0, 0, 1)' : getColumnColor();
```